pages/common/less.md

```diff
@@ -18,10 +18,6 @@
 
 `/{{something}}`
 
-- Backward search for a string (press `n`/`N` to go to next/previous match):
-
-`?{{something}}`
-
```

Comment:
Actually, I think `?` is useful sometimes. @annashorthead Can you merge this example into the `Forward search` example? If so, I think it is ok to merge.

Reply:
Hrm, good point. I'm not sure merging into the previous example would work very well though. Is there an alternative that does a search of the entire document, instead of just backwards / forwards? Maybe we could use that in the example above instead.